src/lib/containers/UserCardListRotate.tsx

```diff
@@ -87,7 +93,15 @@ const UserCardListRotate: React.FunctionComponent<UserCardListRotateProps> = ({s
 
   return (
     <div className="UserCardListRotate">
-      {userIds.length && <UserCardList list={userIds} size={LARGE_USER_CARD} />}
+      {isLoading && loadingScreen}
+      {!isLoading && userIds.length && <UserCardList list={userIds} size={LARGE_USER_CARD} />}
+      <div className="UserCardListRotate__summary">
+        <p>
+          <a className="UserCardListRotate__summary__link btn homepage-button" href="/Explore/People">
```

Comment:
If we could change the href to take in a prop instead. I'm sure that /Explore/People will probably always be the link this goes too, but just so we have the option in the future.

Reply:
Thanks, this has been updated.